packages/elements/src/components/DropdownSelect/index.tsx

```diff
@@ -54,6 +56,7 @@ export const DropdownSelect: React.FC<DropdownSelectProps> = ({
             return (
               <div className="field field-dropdown-select">
                 <label className={`label ${required ? 'required-label' : ''}`}>{labelText}</label>
+                <label className="subtext mb-2">{subText}</label>
```

Comment:
What happens when subtext is not supplied? Could break spacing because there's an empty div with a margin on it